<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTaskEdit</class>
 <widget class="QDialog" name="DialogTaskEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageTask">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QComboBox" name="editProduct">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QComboBox" name="boxAction">
         <item>
          <property name="text">
           <string>Разработка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Доработка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Тестирование</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Командировка</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Сдача в архив</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Другое...</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Изделие:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="editProject">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Действие:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Результат:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Описание:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Проект:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="editDescription"/>
       </item>
       <item row="11" column="0">
        <widget class="QComboBox" name="boxResult">
         <item>
          <property name="text">
           <string>Выполнено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Выполнена часть</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Возникли трудности</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageVacation"/>
     <widget class="QWidget" name="pageIllness"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="boxType">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Задача</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отпуск/отгул</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Больничный</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="editHoursSpent">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="suffix">
        <string> ч.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>168</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutHourPresets"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Время:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTaskEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTaskEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
